src/main/java/com/parasoft/demoapp/service/GlobalPreferencesService.java

```diff
@@ -562,4 +567,13 @@ public void validateKafkaBrokerUrl() throws KafkaServerIsNotAvailableException {
             client.close();
         }
     }
+
+    public void validateRabbitMQServerUrl() throws Exception {
+        Connection connection = rabbitMQConfig.connection();
```

Comment:
Please help to change the `connection` bean inside `RabbitMQConfig `to be connection factory and try to establish connection in here.